experimental/LieAlgebras/src/WeylGroup.jl

```diff
@@ -597,6 +597,29 @@ end
 ###############################################################################
 # G-set functionality (can get moved to src/ once `isomorphism(PermGroup, ::WeylGroup)` gets moved)
 
+"""
+    natural_gset(W::WeylGroup)
+
+Return the G-set `Omega` that consists of the closure of root space elements
+under the natural action of `W`.
+
+!!! warning
+    Currently the seeds are the minimal roots of the root system.
+    This may only be sane for finite types and there may be changes for infinite root systems in the future.
+
+# Examples
+```jldoctest
+julia> W = weyl_group(:A, 2);
+
+julia> length(natural_gset(W))
+6
+```
+"""
+function natural_gset(W::WeylGroup)
+  @req is_finite(W) "Weyl group is not finite"
+  return gset(W, *, roots(root_system(W)); closed=true)
```

Comment:
```suggestion
  return gset(W, roots(root_system(W)); closed=true)
```
without the action function, this routes via `gset_by_type` and thus also providing `check=false` which we want here